src/db.rs

```diff
@@ -307,5 +307,13 @@ CREATE UNIQUE INDEX jobs_name_scheduled_at_unique_index
     ON jobs (
         name, scheduled_at
     );
+",
+    "
+CREATE table review_prefs (
```

Comment:
Is the intention that this will eventually contain review preferences too? (That's what the name suggests)

My first thought was that there should instead be a table of PRs, with assigned reviewers in that list. But, maybe that doesn't make a bunch of sense.

Reply:
Correct. The new DB table at the end of the story will also persist the review preferences for each reviewer (thus the name). I'm trying to slip in changes one by one and as slowly as possible, as per the plan detailed in #1753

PRs in a separate joined table def. make sense in general. But in this case I think the use case is so minimal to not warrant that. _If_ in the far future the tool develops such as to make it a compelling choice, we can always migrate.